frontend/src/conversation/conversationApi.ts

```diff
@@ -1,28 +1,47 @@
 import { Answer } from './conversationModels';
+import { Doc } from '../preferences/preferenceApi';
 
 export function fetchAnswerApi(
   question: string,
   apiKey: string,
+  selectedDocs: Doc,
 ): Promise<Answer> {
-  // a mock answer generator, this is going to be replaced with real http call
+  let namePath = selectedDocs.name;
+  if (selectedDocs.language === namePath) {
+    namePath = '.project';
+  }
+
+  const docPath =
+    selectedDocs.language +
+    '/' +
+    namePath +
+    '/' +
+    selectedDocs.version +
+    '/' +
+    selectedDocs.model;
   return new Promise((resolve, reject) => {
-    setTimeout(() => {
-      let result = '';
-      const characters =
-        'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
-      const charactersLength = characters.length;
-      let counter = 0;
-      while (counter < 5) {
-        result += characters.charAt(
-          Math.floor(Math.random() * charactersLength),
-        );
-        counter += 1;
-      }
-      const randNum = getRandomInt(0, 10);
-      randNum < 5
-        ? reject()
-        : resolve({ answer: result, query: question, result });
-    }, 3000);
+    const activeDocs = 'default';
+    fetch('https://docsgpt.arc53.com/api/answer', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        question: question,
+        api_key: apiKey,
+        embeddings_key: apiKey,
+        history: localStorage.getItem('chatHistory'),
```

Comment:
is this line going to be replaced with something else?

Reply:
In future - yes